modules/nf-core/cellranger/mkfastq/main.nf

```diff
@@ -14,18 +14,18 @@ process CELLRANGER_MKFASTQ {
     path csv
 
     output:
-    path "versions.yml", emit: versions
-    path "${bcl.getSimpleName()}/outs/fastq_path/*.fastq.gz"  , emit: fastq
+    path "${bcl.getSimpleName()}/outs/fastq_path/*.fastq.gz", emit: fastq
```

Comment:
Is there a reason why you're not using `meta.id` (rather than `.getSimpleName`)

Reply:
I didn't create the `cellranger/mkfastq` module, but I will update it now

Reply:
in case it matters @jfy133 , it doesn't look like metadata are passed into this module (no `meta` map is used here)

that might explain why `$meta.id` doesn't appear in the code

Reply:
ok after some thought it seems unwise to change this since it's unclear if `meta` is even necessary here

it seems like this module is usually run prior to executing any nf-core workflows anyways

to wit, nf-core/scrnaseq with the Cellranger aligner starts from FASTQs, not from BCLs